dev/run-frontend-e2e.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+echo -e "\n\n\033[0;33msg start enterprise-e2e must be already running in a different terminal!\033[0m" 
+echo -e "If you run into authentication issues, you can run the following commands to fix them:"
+echo -e "  sg db reset-pg && sg db add-user -name test@sourcegraph.com -password supersecurepassword\n\n"
+
+if [ -z "$GH_TOKEN" ]; then 
+  GH_TOKEN=$(gcloud secrets versions access latest --project=sourcegraph-ci --secret="BUILDKITE_GITHUBDOTCOM_TOKEN" 2>/dev/null)
+  if [ "$GH_TOKEN" == "" ]; then 
+    echo -e "GH_TOKEN is not set, please fetch it from 1Password under the key BUILDKITE_GITHUBDOTCOM_TOKEN" 
```

Comment:
I couldn't find `BUILDKITE_GITHUBDOTCOM_TOKEN` in the shared 1Password vault, but `GITHUB_TOKEN` from the `E2E Pipeline Secrets` note works. Should we update either the 1Password note or this message? Or am I looking at the wrong place 🙂

```suggestion
    echo -e "GH_TOKEN is not set, please fetch it from the "E2E Pipeline Secrets" 1Password note under the key GITHUB_TOKEN" 
```

Reply:
I added it, I forgot to do it earlier :)